src/components/todo/TodoComponent.tsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import { useQueryClient } from '@tanstack/react-query';
-import { useUpdateTodoTime } from '@/app/apiTest/hooks/todoQueries';
+import { useUpdateTodoTime } from '@/api/hooks/todoHooks';
 import * as S from './Todo.styled';
 import { Text } from '../common';
 
 interface TodoComponentProps {
   todoId: number;
   title: string;
-  setTodoId: (s: string) => void;
+  setTodoId: (s: number) => void;
```

Comment:
```suggestion
  setTodoId: (todoID: number) => void;
```
p4: 정말 사소한 거지만 이름을 맞춰주는것이 조금은 더 예쁠 것 같아요!

Reply:
동의합니다:) 반영할게요!